webapp/tbapi/setup.py

```diff
@@ -5,14 +5,15 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'Flask==0.12.2',
-        'Flask-SQLAlchemy==2.3.2',
-        'Flask-WTF==0.14.2',
+        'Flask',
```

Comment:
We should probably `pip freeze` these into a `requirements.txt` file to pin versions
[Some good discussion on `setup.py` vs `requirements.txt`](https://caremad.io/posts/2013/07/setup-vs-requirement/#developing-reusable-things-or-how-not-to-repeat-yourself)